instagram-app/src/components/Login/Login.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+class Login extends React.Component {
+
+  toggle = e => {
+    e.preventDefault();
+    if (localStorage.getItem("loggedIn")) {
+      localStorage.removeItem("loggedIn");
+      // check if loggedIn is in local storage, if so, take it out
+      //and set it to false
+
+    } else {
+      localStorage.setItem("loggedIn", true);
+      // passing in key name, value
+      // otherwise, set it to true if its false
```

Comment:
This should be being done not in local storage but setting a state of logged in from false to true. You can then determine which item you are rendering using a condition based on that state.